packages/fxa-payments-server/src/routes/Product/SubscriptionRedirect/index.tsx

```diff
@@ -30,7 +31,12 @@ export const SubscriptionRedirect = ({
     <div className="product-payment" data-testid="subscription-redirect">
       <div className="subscription-ready">
         <h2>Your subscription is ready</h2>
-        <img alt="Firefox Private Network" src={fpnImage} />
+        <img
```

Comment:
Where the product image gets used in the subscription redirect - visible in local storybook here:

http://localhost:6006/?path=/story/routes-product--success

Reply:
We might need to setup our dev server to allow images from placekitten. Otherwise this guy doesn't load: 
<img width="988" alt="image" src="https://user-images.githubusercontent.com/13018240/69655881-1a638280-103d-11ea-9584-76c45313c54f.png">